### PR TITLE
test case for string conversion

### PR DESCRIPTION
**Pull request by pavelvasev**

### CODE REVIEW

File: tests/QMLEngine/properties.js [hunk 1]
```diff
@@ -27,4 +27,22 @@ describe('QMLEngine.properties', function() {
...
+  /* in Qml, when assigning non-string value to string property,
+     is convert's new value to string. */
+  it('StringConversion', function() {
+    var div = loader('StringConversion');
+    var qml = div.qml;
+
+    expect(qml.stringA).toBe("10");
+    expect(typeof qml.stringA).toBe("string");
+
+    // console.log( "qml.stringB=",qml.stringB,typeof qml.stringB);
```
Comment (ChALkeR) [inline tests/QMLEngine/properties.js]: Commented out debug `console.log` is not needed here.

File: tests/QMLEngine/qml/PropertiesStringConversion.qml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  
+  Column {
+    Text {
+      text: stringA + typeof(stringA)
+    }
+    Button {
+      onClicked: reassign()
+      text: "Reassign"
+    }
+  }
```
Comment (ChALkeR) [inline tests/QMLEngine/qml/PropertiesStringConversion.qml]: Are the Column, Button, and the Text content somehow used/checked in this test?

**Comment by ChALkeR** [comment]
Not sure why these are needed:
```js
    // console.log( "qml.stringB=",qml.stringB,typeof qml.stringB);
```

```qml
  Column {
    Text {
      text: stringA + typeof(stringA)
    }
    Button {
      onClicked: reassign()
      text: "Reassign"
    }
  }
```

Otherwise, LTGM.

**Comment by ChALkeR** [comment]
Ah, I understand that the Text/Button are required to check how Qt behaves.
What about the comment?

**Comment by pavelvasev** [comment]
> Ah, I understand that the Text/Button are required to check how Qt behaves.

Yes I used this to check how qmlscene.exe of 5.x working on these files.

> Not sure why these are needed:
> // console.log( "qml.stringB=",qml.stringB,typeof qml.stringB);

For ongoing debug for someone who will fix this...

**Comment by ChALkeR** [comment]
LGTM, will rebase and merge.

**Pull request merged**